packages/grid/seaweedfs/s3config.json

```diff
@@ -0,0 +1,18 @@
+{
+  "identities": [
+    {
+      "name": "guest",
+      "actions": ["Read"]
+    },
+    {
+      "name": "admin",
+      "credentials": [
+        {
+          "accessKey": "admin",
+          "secretKey": "admin"
```

Comment:
I think in grid/core/config.py
https://github.com/OpenMined/PySyft/blob/d004b6127fa0b37aa0d81c573a1299a7981bf897/packages/grid/backend/grid/core/config.py#L97
The password was set to None, should we change it to "admin"

Reply:
Fixed by [b0c2b88](https://github.com/OpenMined/PySyft/pull/6337/commits/b0c2b8871940bc8b36d6cc41cd103b36dcb4fadd)